backstage/backstage
# [TechDocs] Adjust theme styles

Signed-off-by: Camila Belo <camilaibs@gmail.com>

## Hey, I just made a Pull Request!

- Fixes https://github.com/backstage/backstage/issues/6289
- Style tweaks as a side effect of these changes:
   - https://github.com/backstage/mkdocs-techdocs-core/pull/50
   - https://github.com/backstage/techdocs-container/pull/29

See below how the page appearance is after changes:

| Light theme | Dark theme |
| ------------ | ----------- |
| ![image](https://user-images.githubusercontent.com/6290749/154261684-babc36ab-e133-4188-964d-ece34bd36ebd.png) |  ![image](https://user-images.githubusercontent.com/6290749/154261776-086753d0-9866-4e93-bd21-0b60e1006ce9.png) |

## How to test?

1. Clone the [mkdocs-techdocs-core](https://github.com/backstage/mkdocs-techdocs-core/pull/50) project
2. Check out to the branch `bump-mkdocs-material`
3. Install the plugin locally:
```sh
pip install -e .
```
4. Make sure you have installed version `0.2.3`:
```sh
pip show mkdocs-techdocs-core
```
5. As we are going to test it locally, you need to have your environment configured like [here](https://github.com/backstage/techdocs-container/blob/main/Dockerfile#L21)

6. Navigate to the backstage repository folder
7. Also check out to the branch `bump-mkdocs-material`
8. Run `techdocs-cli `locally in development mode:
```sh
yarn techdocs-cli:dev serve --no-docker
```
9. Start the local embedded app in another terminal:
```sh
yarn workspace techdocs-cli-embedded-app start
```
10. Finally access the docs page:
http://localhost:3000/docs/default/component/local


#### :heavy_check_mark: Checklist
